app/views/admin/friends/_basic_form_fields.html.erb

```diff
@@ -46,6 +46,13 @@
   </div>
 </div>
 
+<div class='row form-group'>
```

Comment:
Could we move :eoir_case_status just below the :status field on this page?

Reply:
commited!